convert_transactions_to_matrix.py

```diff
@@ -0,0 +1,29 @@
+import numpy
+import pandas as pd
+
+def convert_transaction_list_to_matrix(transaction_list):
+    date_vec = []
```

Comment:
If you want, this can be written as:

```python
date_vec = [transaction.date for transaction in transactions]
```

or

```python

from operator import attrgetter

date_vec = list(map(attrgetter('date'), transaction_list))
```

Or you can write everything as

```python
columns = ('date', 'merchant', 'amount', ...)
vectors = list(zip(*map(attrgetter(columns), transaction_list)))
transaction_matrix = numpy.matrix(vectors)
transaction_matrix = transaction_matrix.T
...
```

Or instead of transposing it, just put them in the correct way in the first place:

```python
columns = ('date', 'merchant', 'amount', ...)
transaction_matrix = numpy.matrix(list(map(attrgetter(columns), transaction_list)))
```